src/adapters/ip_camera/api.rs

```diff
@@ -18,6 +19,24 @@ use std::path::Path;
 static CAMERA_USERNAME: &'static str = "admin";
 static CAMERA_PASSWORD: &'static str = "password";
 
+pub fn create_service_id(service_id: &str) -> Id<ServiceId> {
+    Id::new(&format!("service:{}@link.mozilla.org", service_id))
+}
+
+pub fn create_setter_id(operation: &str, service_id: &str) -> Id<Setter> {
+    create_io_mechanism_id("setter", operation, service_id)
+}
+
+pub fn create_getter_id(operation: &str, service_id: &str) -> Id<Getter> {
+    create_io_mechanism_id("getter", operation, service_id)
+}
+
+pub fn create_io_mechanism_id<IO>(prefix: &str, operation: &str, service_id: &str) -> Id<IO>
+    where IO: IOMechanism
+{
+    Id::new(&format!("{}:{}.{}@link.mozilla.org", prefix, operation, service_id))
```

Comment:
I really dislike the @link.mozilla.org stuff on the IDs. As far as I'm concerned this is just more noise and completely unnecessary since the UDNs are already globally unique